projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_checkbox.scss

```diff
@@ -30,3 +30,26 @@ $_dark-fluent-checkbox: extend($_fluent-checkbox, (
 /// @requires extend
 /// @requires $_bootstrap-checkbox
 $_dark-bootstrap-checkbox: extend($_bootstrap-checkbox);
+
+/// Generates a dark indigo checkbox schema.
+/// @type {Map}
+/// @prop {Color} tick-color [#fff] - The checked mark color.
```

Comment:
```suggestion
/// @prop {Color} fill-color [#fff] - The checked mark color.
```

Reply:
You should also add documentation for the rest of the properties you are changing.